Workaround missing capnp-generated header dependency

The generated ninja build system files do not model the dependency that
generates the *.capnp.h headers, and with sufficient parallelism this
causes the build to fail when building the "bl" tool.

Explicitly trigger generation of the header files before building bl to
avoid this.

Upstream-Status: Pending [hack]

--- bin/create_default_bundles_tbz.orig	2020-07-21 15:54:02.000000000 +0200
+++ bin/create_default_bundles_tbz	2020-07-21 15:54:48.000000000 +0200
@@ -14,6 +14,10 @@
 builddir="${builddir:-$HOME/build/TextMate}"
 bl="$builddir/Applications/bl/bl"
 
+ninja -v \
+	Frameworks/plist/src/cache.capnp.h \
+	Frameworks/encoding/src/frequencies.capnp.h
+
 { [ -x "$bl" ] || ninja "$bl"; } && \
 	mkdir -p "$BUNDLES_DST/Managed" && \
 	"$bl" -C "$BUNDLES_DST/Managed" install mandatories defaults && \
